tests/bootstrap.php

```diff
@@ -1,9 +1,20 @@
 <?php
 
 use MongoDB\Tests\Comparator\Int64Comparator;
+use MongoDB\Tests\TestCase;
 use SebastianBergmann\Comparator\Factory as ComparatorFactory;
 
 require __DIR__ . '/../vendor/autoload.php';
 
 // Register custom comparators
 ComparatorFactory::getInstance()->register(new Int64Comparator());
+
+/* Ugly workaround for event system changes in PHPUnit
+ * PHPUnit 10 introduces a new event system, and at the same time removes the
+ * event system present in previous versions. This makes it near impossible to
+ * support PHPUnit 8.5 (required for PHP 7.2) and PHPUnit 9 (PHP < 8.1)
+ * alongside PHPUnit 10. For this reason, we use this bootstrap file to print
+ * test configuration summary.
+ * @todo PHP_VERSION_ID >= 80100: Remove this and use an extension
+ */
+TestCase::printConfiguration();
```

Comment:
This looks like it's only necessary for debugging. How is the "event system" (first I've heard of that term with respect to PHPUnit) relevant to us? AFAIK, we weren't printing any configuration options previously.

Reply:
Unfortunately, there is a hard break with those systems: the existing listener system is removed in PHPUnit 10, but the new event system is only introduced in PHPUnit 10. In order to avoid supporting both versions I decided to opt for this simple variant. From what I've seen, symfony/phpunit-bridge does not have a compatibility layer in place for this change.

Once we use PHPUnit 10 exclusively (IIRC once dropping PHP < 8.1) we can use an extension to print this automatically and remove the workaround.